website_menu_icon/static/src/xml/website.contentMenu.xml

```diff
@@ -0,0 +1,149 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates id="template" xml:space="preserve">
+    <div t-name="website.contentMenu.entry.dialog.icon" t-extend="wysiwyg.widgets.link">
+        <t t-jquery="form" t-operation="append">
+            <div class="form-group row">
+                <label
+                    class="col-form-label col-md-3"
+                    for="o_icon_dialog_label_input"
+                >Icon / Image</label>
+                <div class="col-md-9 input-group ml-auto">
+                    <input
```

Comment:
It would be way more friendly to display a thumbnail of the chosen icon/image than the text of its path or class.